docs/reference/commandline/images.md

```diff
@@ -52,6 +52,39 @@ uses up the `VIRTUAL SIZE` listed only once.
     postgres                  9.3.5               746b819f315e        4 days ago          213.4 MB
     postgres                  latest              746b819f315e        4 days ago          213.4 MB
 
+### Listing images by name and tag
+
```

Comment:
> Finally, there is no future in technical writing --- :-D or rarely.

It took me a moment to interpret that correctly. It sounded a bit "end of an era" 
😄